code/controllers/subsystems/mapping.dm

```diff
@@ -32,6 +32,17 @@ SUBSYSTEM_DEF(mapping)
 		map_templates[T.name] = T
 	preloadBlacklistableTemplates()
 
+/datum/controller/subsystem/mapping/proc/includeTemplate(var/datum/map_template/map_template, var/list/banned_maps)
+	if(!initial(map_template.id))
+		return FALSE
+	var/datum/map_template/MT = new map_template()
+	if(banned_maps)
+		for(var/mappath in MT.mappaths)
+			if(banned_maps.Find(mappath))
+				return FALSE
+	map_templates[MT.name] = MT
+	. = MT
```

Comment:
This proc is mixing returning reference and value types (map template and `FALSE`)
Should either return the map template and null or `TRUE`/`FALSE`

Reply:
Removed `FALSE` return values on earlier returns.